esmvaltool/cmor/_fixes/CMIP6/BCC_ESM1.py

```diff
@@ -0,0 +1,65 @@
+# pylint: disable=invalid-name, no-self-use, too-few-public-methods
+"""Fixes for BCC-ESM1."""
+import numpy as np
+import iris
+
+from ..fix import Fix
+
+
+class allvars(Fix):
+    """Common fixes to all vars"""
+
+    def fix_metadata(self, cubes):
+        """
+        Fix metadata.
+
+        Fixes error in time coordinate, sometimes contains trailing zeros
+
+        Parameters
+        ----------
+        cube: iris.cube.CubeList
+
+        Returns
+        -------
+        iris.cube.CubeList
+
+        """
+        for cube in cubes:
+            try:
+                old_time = cube.coord('time')
+                if old_time.is_monotonic():
+                    pass
+
+                time_units = old_time.units
+                time_data = old_time.points
+
+                idx_zeros = np.where(time_data == 0.0)[0]
+                time_diff = time_units.num2date(time_data[1]) \
+                    - time_units.num2date(time_data[0])
+                days = time_diff.days
+
+                for idx in idx_zeros:
+                    if idx == 0:
+                        continue
+                    correct_time = time_units.num2date(time_data[idx - 1])
+                    if days <= 31 and days >= 28:  # assume monthly time steps
+                        new_time = \
```

Comment:
avoid using backslashes

Reply:
Not sure how to do this in an easy for humans to read way given the 80 character limit. Suggestions welcome. Not sure why this is a problem anyways.